src/ReservoirPair.sol

```diff
@@ -135,8 +135,11 @@ abstract contract ReservoirPair is IAssetManagedPair, ReservoirERC20 {
         _writeSlot0Timestamp(sSlot0, aBlockTimestampLast, false);
     }
 
-    // update reserves with new balances
-    // on the first call per block, update price and liq oracle using previous reserves
+    /// @notice update reserves with new balances
+    /// @notice on the first call per block, accumulate price oracle using previous instant prices and write the new instant prices
+    /// @dev we write an oracle sample even at the time of pair creation. Also we do not update instant prices for
+    /// subsequent mint/burn/swaps in the same block. the team has assessed that this is a small risk given the very
+    /// fast block times on L2s and has decided to make the tradeoff to minimize complexity
```

Comment:
Would typically use proper grammar for natspec (capitalize first letter). Additionally, would say that this comment does not provide much comfort as it says we are taking a known risk instead of explaining why this is fine. Would suggest wording along the lines of "The price is not updated on subsequent stops as manipulating the instaneous price does not materially affect the TWAP, especially when using the clamped pricing."

Reply:
okay updated this comment as well as other NatSpec comments introduced in this PR to have period at the beginning and to end with a period.